spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/web/client/HttpClientBeanPostProcessor.java

```diff
@@ -182,6 +182,7 @@ static void parseConnectionAddress(Connection connection, Span span) {
 
 		@Override
 		public void accept(HttpClientResponse response, Connection connection) {
+			// TODO: is there a way to read the request at response time?
```

Comment:
@violetagg is there any way to do this at the moment or some future release of reactor-netty? The intent is to see the request that actually caused the response. The original request may have been rewritten due to redirects etc.